src/components/List.jsx

```diff
@@ -1,23 +1,54 @@
 import React from 'react';
 import db from '../lib/firebase';
-import { collection } from 'firebase/firestore';
+import { collection, doc, updateDoc } from 'firebase/firestore';
 import { useCollection } from 'react-firebase-hooks/firestore';
+import moment from 'moment';
 
 export const List = ({ token }) => {
   const [value, loading, error] = useCollection(collection(db, token), {
     snapshotListenOptions: { includeMetadataChanges: true },
   });
 
+  const calcTimeDiff = (purchasedTime) => {
+    const timeNow = moment().format();
+    const date1 = moment(timeNow, 'YYYYMMDD HH:mm:ss');
```

Comment:
We can remove this line if we rethink line 13 a bit. `timeNow` is already a moment object if we just remove the use of `.format()`